projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -458,6 +462,98 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 	}
 }
 
+/**
+ * Set the Calypso preference for rdv.
+ *
+ * This is needed to override the ExPlat variation assignment in order to be able to revert the variation for some users.
+ *
+ * @param string|null $assignment The experiment variation.
+ * @return void
+ */
+function wpcom_set_remove_duplicate_views_calypso_preference( $assignment ) {
```

Comment:
The function name is misleading. Are we setting or removing the preference?

Reply:
I guess I can rename it to something like wpcom_set_rdv_calypso_preference